template/src/App.tsx

```diff
@@ -1,11 +1,11 @@
-import 'react-native-gesture-handler';
-import '@localization/i18n';
-import React from 'react';
-import {NavigationContainer} from '@react-navigation/native';
-import RootStackNavigator from '@navigation/navigators/RootStackNavigator';
-import {SafeAreaProvider} from 'react-native-safe-area-context';
-import StoreProvider from 'providers/StoreProvider';
-import SplashScreen from 'react-native-splash-screen';
+import 'react-native-gesture-handler'
+import '@localization/i18n'
+import React from 'react'
+import { NavigationContainer } from '@react-navigation/native'
+import RootStackNavigator from '@navigation/navigators/RootStackNavigator'
+import { SafeAreaProvider } from 'react-native-safe-area-context'
+import StoreProvider from 'providers/StoreProvider'
+import SplashScreen from 'react-native-splash-screen'
```

Comment:
It seems the imports aren't sorted by `prettier`. Can you check why?

Reply:
Yep, fixed, prettier wasn't even working)